### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by dimensi0n**
Related to `hacktoberfest` issuee

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+### Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at [INSERT EMAIL ADDRESS]. All
```
Comment (icco) [inline CONTRIBUTING.md]: Could you change this to `<nat@natwelch.com>`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+## Pull Request Process
+
+1. Ensure any install or build dependencies are removed before the end of the layer when doing a 
+   build.
+2. Update the README.md with details of changes to the interface, this includes new environment 
+   variables, exposed ports, useful file locations and container parameters.
+3. Increase the version numbers in any examples files and the README.md to the new version that this
```
Comment (icco) [inline CONTRIBUTING.md]: You can delete step 3, we always build from head.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue,
+email, or any other method with the owners of this repository before making a change.
```
Comment (icco) [inline CONTRIBUTING.md]: Delete lines 3 + 4 here. I'm pro random pull requests.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+## Pull Request Process
+
+1. Ensure any install or build dependencies are removed before the end of the layer when doing a 
+   build.
+2. Update the README.md with details of changes to the interface, this includes new environment 
+   variables, exposed ports, useful file locations and container parameters.
+3. Increase the version numbers in any examples files and the README.md to the new version that this
+   Pull Request would represent. The versioning scheme we use is [SemVer](http://semver.org/).
+4. You may merge the Pull Request in once you have the sign-off of two other developers, or if you 
+   do not have permission to do that, you may request the second reviewer to merge it for you.
```
Comment (icco) [inline CONTRIBUTING.md]: This step should be reworded: `Please assign @icco to review, he will merge post review approval`

**Comment by icco** [comment]
This is great, just some small tweaks. If you want another PR, you could copy this to https://github.com/icco/graphql/blob/master/CONTRIBUTING.md, because the one you made is much nicer.

**Comment by dimensi0n** [comment]
Ok I'll do that

**Comment by dimensi0n** [comment]
Edited

**Comment by icco** [comment]
Thanks!

**Comment by dimensi0n** [comment]
❤️

**Pull request merged**